android-sdk/templates/android-sdk-persistent-template.j2

```diff
@@ -132,7 +132,7 @@
                   }
                 ],
                 "terminationMessagePath": "/dev/termination-log",
-                "imagePullPolicy": "IfNotPresent",
+                "imagePullPolicy": "Always",
```

Comment:
If we continue to use the latest tag I think that we need to have this set. Otherwise the user would not pull a new image if there was an existing one on the node.